### PR TITLE
Adds achievements for traitor final objectives

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds four new achievements for traitor final objectives:
Stations and Dragons - for the space dragon objective.
The Man behind The Outbreak - for the romerol objective.
Unstable - for the supermatter cascade objective.
Battlecruiser Operational - for the syndicate battlecruiser objective. Also this one is a funny reference.

Additionaly it makes that destabilizating crystal can only be attached to the SM only by traitors, so non-traitors will not get the objective, and will not be able to fucking destroy the station for fun.

(Opened as a draft because i need to get somewhere icons for the achievements, since i am a horrible spriter)

## Why It's Good For The Game

More achievements and more motivation to do final objectives as a tator

## Changelog

:cl: SuperSlayer
add: Added four new achievements for completing final objectives as a traitor
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
let non-traitors do the thing, we have other antagonists and you never know what crazy stuff the sandbox will do where a non-traitor actually can do it within the rules. if you want though you can put in a huge admin notice when a non-traitor is doing it

**Comment by TymurShatillo** [comment]
> let non-traitors do the thing, we have other antagonists and you never know what crazy stuff the sandbox will do where a non-traitor actually can do it within the rules. if you want though you can put in a huge admin notice when a non-traitor is doing it

Okay

**Comment by B00tz** [comment]
Always think we need more stuff to grind on the highscore board.

**Comment by Shadowflame909** [comment]
Yeah there could be mind control afoot. Or black mail

**Comment by rasepretep** [comment]
I'm surprised you didn't name the SM achievement "Unforseen Consequences". And Patient Zero for romerol.

File: code/modules/power/supermatter/supermatter_hit_procs.dm [hunk 1]
```diff
@@ -95,17 +95,22 @@
 		if(get_integrity_percent() < SUPERMATTER_CASCADE_PERCENT)
 			to_chat(user, span_warning("You can only apply \the [destabilizing_crystal] to \a [name] that is at least [SUPERMATTER_CASCADE_PERCENT]% intact."))
 			return
+		
+		var/additional_admin_info = ""		
+		if(!user.mind?.has_antag_datum(/datum/antagonist))
+			additional_admin_info = " as a non-antagonist"
```
Comment (Mothblocks) [inline code/modules/power/supermatter/supermatter_hit_procs.dm:101]: Bunch of random whitespace at the end here

File: code/modules/power/supermatter/supermatter_hit_procs.dm [hunk 2]
```diff
@@ -95,17 +95,22 @@
 		if(get_integrity_percent() < SUPERMATTER_CASCADE_PERCENT)
 			to_chat(user, span_warning("You can only apply \the [destabilizing_crystal] to \a [name] that is at least [SUPERMATTER_CASCADE_PERCENT]% intact."))
 			return
+		
+		var/additional_admin_info = ""
```
Comment (Mothblocks) [inline code/modules/power/supermatter/supermatter_hit_procs.dm:99]: Bunch of random whitespace at the end here

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Pull request closed without merging**